docs/install.md

```diff
@@ -74,5 +74,9 @@ djinni-support-lib/1.0.0
 | ------ | ------ | ----------- |
 | `shared` | `True`, `False` | Wether to build as shared library. Default: `False` |
 | `fPIC` | `True`, `False` | Default: `True`. Follows the default Conan behaviour. |
-| `target` | `jni`, `objc`, `python`, `cppcli`, `auto` | The library has different targets for usage with Java, Objective-C, Python or C++/CLI. By default (`auto`) the target is determined automatically depending on the target OS (`iOS` → `objc`, `Android` → `jni`, `Windows` → `cppcli`). Set this explicitly if you want to build for `macOS`/`Windows`/`Linux`, because on these platforms multiple targets may be a valid option! |
+| `target` | `deprecated` | *Do not use the target option anymore! It exists just to stay compatible to old versions of the recipe and will be removed.* |
+| `with_jni` | `True, False, auto` | Default: `auto`. Wether to build JNI bindings or not. Auto means True for Android builds, False for other platforms. |
+| `with_objc` | `True, False, auto` | Default: `auto`. Wether to build Objective-C bindings or not. Auto means True for any MacOS build, False false other platforms. |
+| `with_python` | `True, False, auto` | Default: `auto`. Wether to build Python bindings or not. Auto means False. |
+| `with_cppcli` | `True, False, auto` | Default: `auto`. Wether to build C++/CLI bindings or not. Auto means True on Windows, False otherwise. |
```

Comment:
```suggestion
| `with_jni` | `True`, `False`, `auto` | Default: `auto`. Wether to build JNI bindings or not. `auto` means `True` for Android builds, `False` for other platforms. |
| `with_objc` | `True`, `False`, `auto` | Default: `auto`. Wether to build Objective-C bindings or not. `auto` means `True` for any Darwin platform (iOS, macOS, ...), `False` false other platforms. |
| `with_python` | `True`, `False`, `auto` | Default: `auto`. Wether to build Python bindings or not. `auto` means `False`. |
| `with_cppcli` | `True`, `False`, `auto` | Default: `auto`. Wether to build C++/CLI bindings or not. `auto` means `True` on Windows, `False` otherwise. |
```

I realize how it's kind of unnecessary to have an `auto` parameter for Python that always defaults to `False` 🙈

Reply:
Python is miss leading anyway, what it should be is C bindings. 
But that will be an extra issue.